src/main/webapp/i18n/de/exam.json

```
@@ -57,7 +51,9 @@
                     "submissionDate": "Eingereicht am: ",
                     "available": "Verfügbar von {{startDate}} bis {{endDate}}",
                     "showMoreAttempts": "Zeige alle Versuche",
-                    "showLessAttempts": "Zeige aktuellste Versuche"
+                    "showLessAttempts": "Zeige aktuellste Versuche",
```

As far as I know there is no superlative for aktuell.
```suggestion
                    "showLessAttempts": "Zeige aktuelle Versuche",
```

---

```suggestion
                    "showLessAttempts": "Zeige aktuellsten Versuche",
```

---

I don't seen an issue with the current translation. 

"Von allen Informationen ist das die aktuellste"
"Von allen Versuchen sind diese vier die aktuellsten"

"Aktuell" is not absolute. Something can be 10 hours old but still "aktuell" or it can already be outdated.
That's different to, for example, "wahr". That one is absolute.

---

This translation was not changed, only the comma was added at the end. If you think there is a mistake, feel free to open a small PR, but I'd like to limit the changes of this PR